### DISCUSSION TITLE
Bazel support

### DISCUSSION THREAD

**Discussion by mnk** [dekorateio/dekorate#826] [Q&A]
Dekorate looks exciting and I would like to try it out.

The root readme.md file states that Dekorate has support for Bazel build, however I can not get it to work.

I have tried adding the [these](https://gist.github.com/mnk/3c2854371a460e8a0c414ecbf5cf43e9) Bazel files to `examples/frameworkless-on-kubernetes-example`, but running `bazel build :lib` fails with the following error:

```
Building liblib.jar (1 source file) and running annotation processors (KubernetesAnnotationProcessor) failed: Worker process did not return a WorkResponse:

---8<---8<--- Start of log, file at /home/mnk/.cache/bazel/_bazel_mnk/46dbaadbce3f025d8355533aafa591a4/bazel-workers/multiplex-worker-7-Javac.log ---8<---8<---
(empty)
---8<---8<--- End of log ---8<---8<---
Target //:lib failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 7.131s, Critical Path: 3.25s
INFO: 7 processes: 2 internal, 5 linux-sandbox.
FAILED: Build did NOT complete successfully
```

Any suggestions on what I am doing wrong?

It would be nice with a Bazel example in the examples folder.

**Comment by iocanel** [comment]
Thanks for sharing your files. Will have a look asap!
